code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -1199,24 +1199,24 @@ GLOBAL_LIST_EMPTY(PDAs)
 		to_chat(user, span_alert("Turn on your receiver in order to send messages."))
 		return
 
-	for (var/obj/item/pda/P in get_viewable_pdas())
-		if (P == src)
+	for (var/obj/item/pda/pda in get_viewable_pdas())
```

Comment:
this should use as anything if get_viewable_pdas() only returns this type which it looks like it does

Reply:
I swear I never get the important notifications. I'll fix that.